app/models/satisfaction_survey_feature_metrics.rb

```diff
@@ -12,13 +12,31 @@ def response_rate(
     success_count / (fail_count + success_count)
   end
 
+  def positive_feedback_rate(
+    start_time,
+    end_time = Time.zone.now.end_of_day
+  )
+    feedback_count = application_forms_with_feedback(start_time, end_time).count.to_f
+    positive_feedback_count = application_forms_with_positive_feedback(start_time, end_time).count.to_f
+    return nil if feedback_count.zero?
```

Comment:
could move this up a line to save a query..? 😄

Reply:
Good idea ;)